salt/roster/terraform.py

```diff
@@ -253,4 +254,4 @@ def targets(tgt, tgt_type="glob", **kwargs):  # pylint: disable=W0613
 
     raw = _parse_state_file(roster_file)
     log.debug("%s hosts in terraform state file", len(raw))
-    return __utils__["roster_matcher.targets"](raw, tgt, tgt_type, "ipv4")
+    return salt.utils.roster_matcher.targets(__opts__, raw, tgt, tgt_type, "ipv4")
```

Comment:
Did you mean to add `__opts__` here?

Reply:
Yes. The utils module _used_ to use `__opts__`.
Now it doesn't, we have to pass them in.